_inc/lib/class.core-rest-api-endpoints.php

```diff
@@ -37,6 +38,8 @@ class Jetpack_Core_Json_Api_Endpoints {
 
 	/**
 	 * @var string Generic error message when user is not allowed to perform an action.
+	 *
+	 * @deprecated 8.8.0 Use `REST_Connector::get_user_permissions_error_msg()` instead.
```

Comment:
Should we use `REST_Connector::get_user_permissions_error_msg()` to set `self::$user_permissions_error_msg` in `register_endpoints()`, so we don't have duplicate code?

Reply:
Done 👍